README.md

```diff
@@ -2,6 +2,7 @@
 
 [![GitHub license](http://dmlc.github.io/img/apache2.svg)](LICENSE.md)
 [![Build Status](https://travis-ci.org/shopgate/connect-integration-sdk.svg?branch=master)](https://travis-ci.org/shopgate/connect-integration-sdk)
```

Comment:
The build status badge needs to point to https://api.travis-ci.org/shopgate/connect-integration-sdk-php.svg, also the travis link needs to be updated.